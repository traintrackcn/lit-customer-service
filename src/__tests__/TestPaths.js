import { fromJS } from 'immutable';
import p from '../rPath';


describe('', () => {

    it('', ()=>{

        // expect(p.appConfig.company).toMatchSnapshot();
        expect(p.appConfig.platform).toMatchSnapshot();
        expect(p.appConfig.category).toMatchSnapshot();
        expect(p.appConfig.value).toMatchSnapshot();
        expect(p.billing.plan).toMatchSnapshot();

    });
    

});