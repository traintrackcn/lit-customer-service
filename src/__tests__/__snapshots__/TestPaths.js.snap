// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
Array [
  "r",
  "appConfig",
  "company",
]
`;

exports[` 2`] = `
Array [
  "r",
  "appConfig",
  "platform",
]
`;

exports[` 3`] = `
Array [
  "r",
  "appConfig",
  "category",
]
`;

exports[` 4`] = `undefined`;
